.docker/entrypoint/lib/extensions/object_extensions.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Object extensions
+class Object
+  def must_be_a_directory!
+    raise ArgumentError, 'Value cannot be nil' if nil?
+
+    must_be_a! String
+
+    raise ArgumentError, 'String cannot be empty' if empty?
+    raise Jekyll::PlantUml::DirectoryNotFoundError, "#{self} does not exist" unless Dir.exist?(self)
+  end
+
+  def must_be_a_file!
+    raise ArgumentError, 'Value cannot be nil' if nil?
+
+    must_be_a! String
+
+    raise ArgumentError, 'String cannot be empty' if empty?
+    raise Jekyll::PlantUml::FileNotFoundError, "#{self} cannot be found." unless writable_file?
```

Comment:
Codacy found an issue: [Line is too long. [95/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)